monai/bundle/scripts.py

```diff
@@ -822,6 +822,59 @@ def verify_metadata(
     logger.info("metadata is verified with no error.")
 
 
+def _get_net_io_info(
+    parser: ConfigParser | None = None, prefix: str = "_meta_#network_data_format", p: int = 1, n: int = 1, any: int = 1
+) -> tuple:
+    """
+    Get the input and output information defined in the metadata.
+
+    Args:
+        parser: a parser of the given bundle.
+        prefix: a prefix for the input and output id, which will be combined as `prefix#input` and
+            `prefix#output` to parse the input and output information in the `metadata.json` file of
+            bundle, default to `meta_#network_data_format`.
+        p: power factor to generate input data shape if dim of expected shape is "x**p", default to 1.
+        n: multiply factor to generate input data shape if dim of expected shape is "x*n", default to 1.
+        any: specified size to generate input data shape if dim of expected shape is "*", default to 1.
+
+    Returns:
+        input_channels: the channel number of the first input.
```

Comment:
it's not the "first" input, just the item with key "image".

Thanks.

Reply:
Ok, thanks. Will update this.

Reply:
I don't see you update these 2 lines.